tt_metal/impl/debug/sanitize_noc_host.hpp

```diff
@@ -9,6 +9,7 @@
 
 namespace tt {
 
+// Host MMIO reads/writes don't have alignment restrictions, so no need to check alignment here.
 #define DEBUG_VALID_L1_ADDR(a, l) (((a) >= MEM_L1_BASE) && ((a) + (l) <= MEM_L1_BASE + MEM_L1_SIZE))
```

Comment:
I was originally thinking about ways to merge the defines here with the ones in `sanitize_noc.h`, since it was a little strange there was a host and device copy of them with slightly different checks. But now they're diverging even more, not sure if it makes sense to keep them like this or not.

Reply:
yeah, eventually we should clean this up some time.  need the alignment restrictions in device specific dirs and have all these in a place where the device can export them and the host can include them.  ok to defer

Reply:
Ok, deferring this for now